composer.json

```diff
@@ -1,10 +1,22 @@
 {
     "name": "omise/omise-magento",
-    "version": "2.3.0-dev",
+    "description": "Accept payments on your Magento 2 website with Omise",
+    "keywords": ["omise", "payment", "debitcard", "creditcard", "magento"],
+    "homepage": "https://github.com/omise/omise-magento",
+    "authors": [
+        {
+          "name": "Omise",
+          "email": "support@omise.co"
+        }
+    ],
+    "version": "2.4.0-dev",
+    "minimum-stability": "stable",
```

Comment:
@jonrandy its default is already `stable`, do we still need this one?

Reply:
I was going for a "belt and braces" approach